doc/lua_api.txt

```diff
@@ -2162,6 +2186,22 @@ Elements
 ### `tabheader[<X>,<Y>;<name>;<caption 1>,<caption 2>,...,<caption n>;<current_tab>;<transparent>;<draw_border>]`
 
 * Show a tab**header** at specific position (ignores formsize)
+* **Important note**: This syntax for tabheaders can only be used with the
+  old coordinate system.
+* `X` and `Y`: position of the tabheader
+* `name` fieldname data is transferred to Lua
+* `caption 1`...: name shown on top of tab
+* `current_tab`: index of selected tab 1...
+* `transparent` (optional): show transparent
+* `draw_border` (optional): draw border
+
+### `tabheader[<X>,<Y>;<H>;<name>;<caption 1>,<caption 2>,...,<caption n>;<current_tab>;<transparent>;<draw_border>]`
+
+* Show a tab**header** at specific position (ignores formsize)
+* **Important note**: This syntax for tabheaders can only be used with the
+  new coordinate system.
+* `X` and `Y`: position of the tabheader
+* `H`: height of the tabheader
```

Comment:
I agree in a way, but width for tabs would be strange since it would apply to all tabs, so text might be cut off on some and have a lot of padding on another.  Another potential solution is to make the width add padding to the sides of the text.

I'd like more input on how I should do this.

Reply:
surely it would be the width of the tab header? So it would stop and add the `<>` at a smaller width, as shown here:

![image](https://user-images.githubusercontent.com/2122943/59316149-e37af980-8cb4-11e9-9e5b-37db89216899.png)

Reply:
Ahh, that makes sense.  I'll implement this then.  I think that it should be allowed to be left as default as well, though, since that's good enough for most people.